### ISSUE TITLE
try_without_catch, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#881]
```abap
    TRY.
        ls_vseoclass = mi_object_oriented_object_fct->get_class_properties( ls_clskey ).
      CLEANUP.
        zcl_abapgit_language=>restore_login_language( ).
    ENDTRY.
```
